libmscore/glissando.cpp

```diff
@@ -323,6 +323,10 @@ void Glissando::layout()
 
       // initial note dots / ledger line / notehead
       offs1 *= -1.0;          // discount changes already applied
+
+      // Accommodate for TAB numerals for x1 position
+      offs1.rx() += (cr1->staff()->isTabStaff(cr1->tick())) ? anchor1->tabHeadWidth() * 0.50 : 0;
```

Comment:
I could check here for making sure the fret numnber is above 9 first like
if (cr1->staff()->isTabStaff(cr1->tick()))
            offs1.rx() += anchor1->fret() > 9 ? anchor1->tabHeadWidth() * 0.50 : 0;

and I think all would be okay, but upon further looking into the default as it is already for 0-9, the offset looks like it could be a little closer to the fret number like with the noteheads... so I think this should be fixed up at the same time rather than merely offsetting for a double-digit fret number only.